docker-compose.yml

```diff
@@ -1,11 +1,62 @@
 version: '3'
 
+networks:
+    node_network:
+        driver: bridge
+
 services:
-    postgres:
-        image: 'postgres:15.2'
+    web:
+        container_name: 'web'
+        build:
+            context: ./
+            dockerfile: ./frontend/Dockerfile
+        environment:
+            - VITE_APP_NODE_ENV=production
+            - VITE_APP_API_ORIGIN_URL=http://localhost:3001/api/v1
+            - VITE_APP_DOMAIN_URL=http://localhost:3000
+        env_file:
+            - ./frontend/.env
+        networks:
+            - node_network
+        ports:
+            - '3000:3000'
+        depends_on:
+            - api
+
+    api:
+        container_name: 'api'
+        build:
+            context: ./
+            dockerfile: ./backend/Dockerfile
+        restart: unless-stopped
+        environment:
+            - NODE_ENV=production
+            - PORT=3001
+            - HOST=0.0.0.0
```

Comment:
is this host Url correct? Api and databaseare running from different containers. You should specify **db** as host.

Reply:
Yes, it is correct. I defined this HOST variable as a server host (api service). You can see that api service connects to DB through url in ENVs 

DB_CONNECTION_STRING=pg://postgres_user:postgres_password@**db**:5432/postgres_db - _here **db** is a host._